ui/apps/portal/src/types/trex.ts

```diff
@@ -1,5 +1,6 @@
 export interface TrexPlugin {
   name: string;
+  description?: string;
```

Comment:
NIT: looking at the trex plugin endpoint, description is always present. so this does not need to be optional